repackage-testvm.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Copy-paste the repackage script with minor changes, so that the base box for the tutorial can be conveniently updated.

Reply:
It is not part of the documentation website.

Reply:
I think a demo (with all developers) of this doc site, and these repackage scripts, is suitable. Maybe during the dev meeting on Thursday?